app/resources/admin_api_key_form.html.template

```diff
@@ -82,6 +104,11 @@
 	Permission flags
       </label>
       <div class="response">
+        <input type="checkbox" name="write_permission" id="write_permission" onChange="updateWriteDomain();">
+        Write permission
+	<span class="warning">This permission should go with domain write permission.</span>
```

Comment:
It would be clearer to say:

    This permission should go with the domain name above.

Actually it may be even better to move the domain name text box just below the "Write permission" checkbox (i.e., above "Read permission" checkbox). But it's up to you. We can do that later.